content/release-notes/release-notes.md

```diff
@@ -17,20 +17,18 @@ For more information see [PI Adapter for OPC UA overview](xref:PIAdapterForOPCUA
 
 ### Fixes
 
-The following issues reported from version 1.1 are fixed in this release.
+The following issues reported from version 1.2 are fixed in this release.
```

Comment:
I'd omit the "reported from version 1.2" part of this phrase.

Reply:
removed